server/config/lib/client.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
 
 module.exports = {
-  baseUri: process.env.DS_ROGUE_API_BASEURI,
+  services: {
+    northstar: {
+      baseUri: process.env.DS_NORTHSTAR_API_BASEURI,
+    },
+    rogue: {
+      baseUri: process.env.DS_ROGUE_API_BASEURI,
+    },
```

Comment:
I could see handling the custom Gambit auth from within this config vs creating files in `authStrategies`:
```
services: {
  northstar: { baseUri: process.env.DS_NORTHSTAR_API_BASEURI },
  rogue: { baseUri: process.env.DS_ROGUE_API_BASEURI  },
  // Would this be better as a gambit object with gambit.content and gambit.conversations? Ideally we want to get rid of Gambit Content altogether with GraphQL
  gambitConversations: {
    baseUri: process.env.DS_GAMBIT_CONVERSATIONS_API_BASEURI,
    apiKey: process.env.DS_GAMBIT_CONVERSATIONS_API_KEY,
  },
  gambitCampaigns: {
    baseUri: process.env.DS_GAMBIT_CAMPAIGNS_API_BASEURI,
    apiKey: process.env.DS_GAMBIT_CAMPAIGNS_API_KEY,
  },
  graphql: { baseUri: process.env.DS_GRAPHQL_BASEURI },
},
```